### PR TITLE
Fix: [Web App] Add (Code Collabo) favicon to community app

### LINKED ISSUE #45: Add (Code Collabo) favicon to community app
#### Issue and expected result
The favicon (i.e. the tiny image beside the page title in the browser tab) that is currently in the app is not our own.
- Reach out to Collabo Community designer @Geefundesign (Isaiah) to know what to use as favicon (please use the appropriate public channel on our discord server for this discussion, not DM).
- Help us change/add the favicon for `Code Collabo` to the app: https://github.com/code-collabo/collabo-community-app

### PR DESCRIPTION
**Pull request by Akpjunior94**
**This pull request makes the following changes**
- Fixes code-collabo/collabo-community-app#45

**General checklist**
- [x] File or folder now contains changes as specified in the issue i worked on
- [x] I have linked the issue I worked on to this pull request submitted by me

**Testing checklist**

Replace this text with the testing checklist from the issue this pull request fixes.

Ping @code-collabo/web-app

### CODE REVIEW

File: client/src/components/shared/PageHeadElement.tsx [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+        type="image/png"
+        sizes="16x16"
+        href="/favicon-16x16.png"
+      />
+      <link rel="manifest" href="/site.webmanifest" />
+      <title>{PageTitle}</title>
     </Head>
   );
-};
+}
```
Comment (Ifycode) [inline client/src/components/shared/PageHeadElement.tsx]: **_Review comment 1_**
@AbrahamAkp please PageHeadElement file contain only the changes you are making. Restore all formatting to what was there previously.

**Review by Ifycode — CHANGES_REQUESTED** [review]
**_Review comment 2_**
@AbrahamAkp help remove the remaining vercel and next images that are in there.

**Comment by Ifycode** [comment]
**_Review comment 3_**
@AbrahamAkp Check out the **_Guidelines for submitting pull requests to any repository_** section of the contributing guide to know how we go about the Title of pull requests: https://docs.collabocommunity.com/collabo-contributor/collabo-contributor-docs/contributing-guide#guidelines-for-submitting-pull-requests-to-any-repository

File: client/src/components/shared/PageHeadElement.tsx (hunk 1 above)
Comment (Akpjunior94) [reply client/src/components/shared/PageHeadElement.tsx]: Sorry i don't understand. the <title>{PageTitle}</title> which is the previous code is still in the component.

Comment (Ifycode) [reply client/src/components/shared/PageHeadElement.tsx]: The latter part - "Restore all formatting to what was there previously". This is the problem.

Comment (Ifycode) [reply client/src/components/shared/PageHeadElement.tsx]: Only the changes you are or should be making is what we are interested in. Any other thing should not change in the file.

Comment (Akpjunior94) [reply client/src/components/shared/PageHeadElement.tsx]: Oh I get now. The issue is the prettier I am using in my VSCode. Saving will definately reformat the code. what do you suggest? Maybe we can setup a global formatter and add to the guidelines as well. I will uninstall my pretttier so it doesn't format it.

**Comment by Akpjunior94** [comment]
> **_Review comment 3_** @AbrahamAkp Check out the **_Guidelines for submitting pull requests to any repository_** section of the contributing guide to know how we go about the Title of pull requests: https://docs.collabocommunity.com/collabo-contributor/collabo-contributor-docs/contributing-guide#guidelines-for-submitting-pull-requests-to-any-repository

done

File: client/src/components/shared/PageHeadElement.tsx (hunk 1 above)
Comment (Ifycode) [reply client/src/components/shared/PageHeadElement.tsx]: I have the idea. We will set it up. For now, you can just do away with your formatter anytime you are making changes to this repository.

Comment (Akpjunior94) [reply client/src/components/shared/PageHeadElement.tsx]: ok

Comment (Akpjunior94) [reply client/src/components/shared/PageHeadElement.tsx]: @Ifycode PR updated.

**Comment by Ifycode** [comment]
Is this ready for review? @Akpjunior94 
Please see my **_review comment 3_** above about your pull request title: https://github.com/code-collabo/collabo-community-app/pull/53#issuecomment-1793507328

**Comment by Akpjunior94** [comment]
> Is this ready for review? @Akpjunior94 Please see my **_review comment 3_** above about your pull request title: [#53 (comment)](https://github.com/code-collabo/collabo-community-app/pull/53#issuecomment-1793507328)

It has been changed.

**Comment by Akpjunior94** [comment]
@Ifycode All reviews has been attended to.

**Comment by Ifycode** [comment]
**_Review comment 3_** has not been effected @Akpjunior94 
Again, pointing you to the contributing guide page of the documentation: https://github.com/code-collabo/collabo-community-app/pull/53#issuecomment-1793507328

**Comment by Akpjunior94** [comment]
> **_Review comment 3_** has not been effected @Akpjunior94 Again, pointing you to the contributing guide page of the documentation: [#53 (comment)](https://github.com/code-collabo/collabo-community-app/pull/53#issuecomment-1793507328)

Please check again.. Corrected the title.

**Review by Ifycode — APPROVED** [review]
Nicely done @Akpjunior94 
Thank you.

**Pull request merged**